docs/getting-started.md

```diff
@@ -306,6 +313,26 @@ bazel run -c dbg --run_under="valgrind --tool=callgrind --callgrind-out-file=/tm
 
 This will output the file at the _absolute_ path given via the `--callgrind-out-file` argument. This file can then be viewed using `kcachegrind` (example: `kcachegrind /tmp/profile.callgrind`), giving lots of useful information about where time is being spent in the code.
 
+Callgrind requires building with debug symbols; Thus, runtime performance will be much slower, so Callgrind is not appropriate to find bottlenecks in real runtime environments.
+
+### Tracy
+
+Tracy is a lightweight, real-time profiler designed for understanding real-time performance of a system. It offers insights into CPU usage and memory allocations by adding Tracy's markup API.
```

Comment:
You should mention that Tracy can be installed with the `environment_setup/install_tracy.sh` script